cnf-certification-test/identifiers/remediation.go

```diff
@@ -79,6 +79,8 @@ const (
 
 	OperatorIsInstalledViaOLMRemediation = `Ensure that your Operator is installed via OLM.`
 
+	OperatorHasSemanticVersioningRemediation = `Ensure that the installed Operator has semantic versioning.`
```

Comment:
```suggestion
	OperatorHasSemanticVersioningRemediation = `Ensure that the installed Operator's  semantic versioning is correct.`
```

Reply:
Following this, I propose something else : `Ensure that the operator's version used semantic versioning.` 
https://sdk.operatorframework.io/docs/best-practices/best-practices/